src/applications/ivc-champva/10-10D/config/form.js

```diff
@@ -624,11 +642,20 @@ const formConfig = {
                 ...titleUI(
                   ({ formData }) =>
                     `${applicantWording(formData)} mailing address`,
+                  ({ formData, formContext }) => {
+                    const txt =
+                      'We’ll send any important information about your application to this address';
+                    // Prefill message conditionally displays based on `certifierRole`
+                    return formContext.pagePerItemIndex === '0' ? (
+                      <>
+                        <p>{txt}</p>
+                        {CustomPrefillMessage(formData, 'applicant')}
+                      </>
+                    ) : (
+                      <p>{txt}</p>
+                    );
+                  },
```

Comment:
Same as all the other uses of `CustomPrefillMessage` - if the signer is also an applicant, we copy their address over to the first applicant slot and show this blue alert.